media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -16,10 +16,7 @@ <h2>
 <p>
 Chcemvediet.sk, občianske združenie so sídlom a adresou pre doručovanie Karpatské námestie 7770/10A,
 831 06 Bratislava, Slovenská republika, IČO: 50436376, zapísané v registri občianskych združení
-vedenom Ministerstvom vnútra SR
-</p>
-<p>
-(ďalej ako „Prevádzkovateľ“)
+vedenom Ministerstvom vnútra SR (ďalej ako „Prevádzkovateľ“)
```

Comment:
@martinmacko47 Aj definicia zakona je v jednom paragrafe, aj mne osobne sa mi to viac paci v jednom paragrafe.

Reply:
Ano, bolo to zle zalomene. Vdaka.